pointmatcher/ErrorMinimizers/PointToPlane.cpp

```diff
@@ -154,7 +192,22 @@ typename PointMatcher<T>::TransformationParameters PointToPlaneErrorMinimizer<T>
 		assert(normalRef.rows() > 0);
 
 		// Compute cross product of cross = cross(reading X normalRef)
-		const Matrix cross = this->crossProduct(mPts.reading.features, normalRef);
+		Matrix cross;
+		Matrix Gamma(3,3);
```

Comment:
Variable doesn't follow camelCasing style

Reply:
Will fix that.